### ISSUE TITLE
refresh-queued forbids more than one element being mounted 

### ISSUE THREAD

**Issue by geraldodev** [weavejester/brutha#1]
Consider app and app2 two valid ids

```
cljs.user=> (def r #js {})
#js {}
cljs.user=> (aset r (.getElementById js/document "app") true)
true
cljs.user=> r
#js {:[object HTMLDivElement] true}
cljs.user=> (aset r (.getElementById js/document "app2") true)
true
cljs.user=> r
#js {:[object HTMLDivElement] true}
```

I have a case like this and was wondering why just the first brutha mounted node worked.

As proof I've changed refresh-queued to

```
(defn mount [element node]
  (when-not (aget refresh-queued (.-id node))
    (aset refresh-queued (.-id node) true)
    (req-anim-frame (fn []
                      (js-delete refresh-queued (.-id node))
                      (js/React.render element node)))))
```

The code is not a suggestion.

**Comment by weavejester** [comment]
Thanks for the bug report. I changed `refresh-queued` into a mutable Javascript object for speed, but forgot that Javascript objects only accept strings as keys.

**Comment by weavejester** [comment]
Looks like an atom containing a set isn't that much slower than a Javascript object, so I'll use that until I can think of something better.

**Comment by weavejester** [comment]
Should be fixed in version 0.1.4.
